skyportal/handlers/api/sources_confirmed_in_gcn.py

```diff
@@ -222,18 +243,14 @@ def post(self, dateobs):
 
         """
         data = self.get_json()
-        try:
-            arrow.get(dateobs).datetime
-        except Exception:
-            return self.error(f'Invalid dateobs: {dateobs}')
         if 'localization_name' not in data:
             return self.error("Missing required parameter: localization_name")
         if 'localization_cumprob' not in data:
             return self.error("Missing required parameter: localization_cumprob")
         if 'source_id' not in data:
             return self.error("Missing source_id")
-        if 'confirmed_or_rejected' not in data:
-            return self.error("Missing confirmed_or_rejected")
+        if 'confirmed' not in data:
+            return self.error("Missing confirmed")
```

Comment:
If you pass `data` directly to the validator it may save you this entire block of code.

Reply:
Done. I decided to use one validation, with added conditional validation (for required / non required values) for the different GET, POST, PATCH, DELETE methods, as they don't required the same parameters.